.github/workflows/ci.yaml

```diff
@@ -73,12 +73,21 @@ jobs:
           - integration-tls
           - integration-azure
           - integration-logs
+        runner:
+          - ubuntu-22.04
+          - [self-hosted, linux, AMD64, X64, large, jammy]
+        exclude:
+          - tox-environments: integration-azure           # Don't test azure twice
```

Comment:
praise: now that we test both self-hosted and github runners, any platform-dependent inconsistencies will now be easier to spot.